pkg/cli/command_test.go

```diff
@@ -136,9 +136,9 @@ func TestCommandsFailExecute(t *testing.T) {
 			args: []string{
 				"diff",
 				"--dir1",
-				filepath.Join(testutils.GetTestsDir(currentPkg), "onlineboutique_workloads"),
+				filepath.Join(testutils.TestsDirPath, "onlineboutique_workloads"),
```

Comment:
Instead of duplicating the code `filepath.Join(testutils.TestsDirPath, "some test dir name")`, export a function from `testutils` that gets the dir name and returns the path for this test.

Reply:
done